openbb_platform/extensions/platform_api/openbb_platform_api/utils/widgets.py

```diff
@@ -241,7 +254,11 @@ def build_json(  # noqa: PLR0912  # pylint: disable=too-many-branches, too-many-
     if not openapi:
         return {}
 
-    starred_list: list = []
+    starred_list: list = [
+        "/api/v1/technical/*",
```

Comment:
Is there a way to avoid hardcoding these?

Reply:
Adding "exclude" entries for every single function is one way, this is just more efficient and there is no way of supporting these endpoints in Workspace currently.  When there is a way to use the output of a widget as the input to a POST endpoint, then we can revisit a way to handle these paths.

Maybe you mean something else though?

Reply:
1. We allow the user to specify the api path in system_settings overriding api/v1 

2. I really want this package to be usable as a generic launcher for workspace backends so this pushes me to question all hardwiring to the existing extensions